en/en-US/Content/0.4.5 - App Settings/appSettings.json

```diff
@@ -3,16 +3,6 @@
 	"appSettings_subtitle": "Customize your Radix Wallet",
 	"appSettings_linkedConnectors_title": "Linked Connectors",
 	"appSettings_gateways_title": "Network Gateways",
-	"appSettings_entityHiding_title": "Account &amp; Persona Hiding",
```

Comment:
are these changes on purpose? We shouldn't be modifying the language specific (in this case, English-US) 
 files ourselves

Reply:
my mistake. Reverted this file to how it was